test_bundle_corrected.py

```
@@ -0,0 +1,70 @@
+__mypackage_DEBUG_MODE = config.DEBUG
+
+# ─ Module: mypackage.utils ─
+helper_function = helper_function
+MAX_ITEMS = MAX_ITEMS
+DEFAULT_VALUE = DEFAULT_VALUE
+"""Utils subpackage with re-exports."""
+__all__ = ["helper_function", "MAX_ITEMS", "DEFAULT_VALUE"]
+
+# ─ Entry Module: main ─
+__main_DEBUG_MODE = config.DEBUG
+print(f"Debug mode: {DEBUG_MODE}")
```

Pylint (3.3.7)

[error] Undefined variable 'DEBUG_MODE'

(E0602)